src/main/java/bio/terra/grammar/azure/SynapseVisitor.java

```diff
@@ -24,11 +25,14 @@ public String visitTable_expr(SQLParser.Table_exprContext ctx) {
     String alias = generateAlias(tableName);
     return """
       OPENROWSET(
-        BULK 'parquet/%s/*/*.parquet',
+        BULK '%s',
         DATA_SOURCE = '%s',
         FORMAT = 'parquet') AS %s
       """
-        .formatted(tableName, sourceDatasetDatasource, alias);
+        .formatted(
+            FolderType.METADATA.getPath("parquet/%s/*/*.parquet".formatted(tableName)),
+            sourceDatasetDatasource,
+            alias);
```

Comment:
Wouldn't we need to specify the top layer container here as well? <snapshot id or dataset id>/metadata/parquet...

Reply:
Ah, it's weird but no because the container is part of the datasource definition now.  So the root here is the container